src/plugins/sockets/inGameEvents.js

```diff
@@ -34,7 +34,7 @@ export async function handleInGameEvents(evData) {
       gameStore.updateReady(evData.pNum);
       return;
     }
-    case SocketEvent.INITIALIZE: {
+    case SocketEvent.DEAL: {
```

Comment:
These should also be constants that we can reuse.

Reply:
Isn't it already? This is imported as a reusable enum. 

That said, the weirdness imo is the heavy overlap with MoveType. They aren't exactly the same because SocketEvent includes some event names that aren't actually GameStates eg conceding, a new rematch game being created. So it feels weird to have them as two separate imports, but MoveType doesn't allow all the values used here, so I'm not sure how to align them. One move would be to have SocketEvent extend MoveType by adding extra properties

Reply:
I don’t even think I noticed it was imported lol, but I agree with the extend idea if it truly is a subset.

Reply:
https://github.com/cuttle-cards/cuttle/issues/1004